tests/smart-tokens.ts

```diff
@@ -5,9 +5,7 @@ export type StateOverride = {
 };
 
 export type StateSimulateApiOverride = {
-  storage: {
-    value: Record<string, string>;
-  };
+  storage: Record<string, string>;
 };
```

Comment:
Why do we need to update this type if we still set overrides to `value` field (in this file on `108`th row)?